docs/getting-started/install-scylla/oss-install-aws.rst

```diff
@@ -0,0 +1,112 @@
+
+#. Configure your instance details. 
+
+   * **Number of instances** – If you are launching more than one instance, make sure to set the seed’s IPs correctly, either in the User Data (below) or after the launch.
+   * **Network** – Configure the network settings.
+      
+     * Select “Launch into EC2-Classic”, or your own VPC.
+     * Configure the security group. Ensure that all :ref:`ScyllaDB ports <networking-ports>` are open.
+
+   * **Advanced Details> User Data** – You can add the following options in the JSON format:
+
+     * ``cluster_name`` - The name of the cluster.
+     * ``experimental`` - If set to ``true``, it enables all experimental features.
+     * ``seed_provider`` - Seeds provider and a list of seeds.
+     * ``developer_mode`` - If set to ``true``, ScyllaDB will run in :doc:`developer mode </getting-started/installation-common/dev-mod>`.
+     * ``post_configuration_script`` - A base64 encoded bash script that will be executed after the configuration is completed.
+     * ``start_scylla_on_first_boot`` - Start ScyllaDB once the configuration is completed.
+
+     For full documentation of ScyllaDB AMI user data, see the `ScyllaDB Image documentation <https://github.com/scylladb/scylla-machine-image>`_.
+
+     Example:
+
+   .. code-block:: console
+
+        {
+        "scylla_yaml": {
+          "cluster_name": "my-test-cluster",
+          "experimental": true
+          }
+        }
+
+#. Add storage.
+
+   * ScyllaDB AMI requires XFS to work. You **must** attach at least one drive for ScyllaDB to use as XFS for the data directory. 
+     When attaching more than one drive, the AMI setup will install RAID0 on all of them.
+   * The ScyllaDB AMI requires at least two instance store volumes. The ScyllaDB data directory will be formatted with XFS when the instance 
+     first boots. ScyllaDB will fail to start if only one volume is configured.
+
+#. Tag your instance.
+#. Click **Launch Cluster**. You now have a running ScyllaDB cluster on EC2.
+
+Launching a Cluster Using CloudFormation
+--------------------------------------------
+
+.. note::
+
+    For evaluation only.
+
+
+#. Go to `Amazon EC2 AMIs – ScyllaDB <https://www.scylladb.com/download/?platform=aws#open-source>`_ in ScyllaDB's download center, 
+   choose your region, and click the **Cluster** link to open the stack creation wizard.
+#. Configure and create the stack.
+
+
+Connecting 
+---------------
+
+Connect to the servers using the username ``scyllaadm``.
+
+.. code:: console
+    
+    ssh -i your-key-pair.pem scyllaadm@ec2-public-ip
+
+The default file paths:
+    
+* The ``scylla.yaml`` file: ``/etc/scylla/scylla.yaml``
+* Data: ``/var/lib/scylla/``
+
+To check that the ScyllaDB server and the JMX component are running, run:
+
+.. code-block:: console
+
+  nodetool status
+  
+.. note::
+
+    This ScyllaDB AMI does not support automatic multi-zone deployment.
```

Comment:
What does this note refer to? I guess it doesn't belong here but should be moved somewhere up.

Reply:
The cloud formation is limited. I sugget to remove it.